app/models/post.rb

```diff
@@ -4,6 +4,8 @@ class Post < ApplicationRecord
 
   has_many :comments
```

Comment:
For `has_many` associations we normally should put `dependent: :destroy` to ensure these associated records get deleted when a `post` record is destroyed